modules/custom/dkan_admin/dkan_admin.module

```diff
@@ -41,18 +41,17 @@ function dkan_admin_preprocess_views_view_field(&$vars) {
   $uuid = $entity->uuid->value;
 
   // To access entities from relationship.
-  // $entities = $vars['row']->_relationship_entities;
-
-  if (isset($vars['view']) 
-    && ($vars['view']->id() == 'dkan_content') 
+  // $entities = $vars['row']->_relationship_entities;.
+  if (isset($vars['view'])
```

Comment:
Consider removing the unused code or at least remove the tailing period added by the coding standards.